C/manual/ch_Finance-Quote.xml

```diff
@@ -435,13 +474,17 @@ SYNOPSIS
           <step>
             <para>Add the following line to your crontab:
               <informalexample>
-<screen>0 16 * * 5 gnucash-cli --quotes get &user-datafile; &gt; /dev/null 2&gt;&amp;1</screen>
+<?dbfo pgwide="1"?>
+<screen>
+  0 16 * * 5 gnucash-cli --quotes get &user-datafile; &gt; /dev/null 2&gt;&amp;1
```

Comment:
No indentation! Check if the newline is ignored in the different output formats.